lib/database.js

```diff
@@ -1,4 +1,4 @@
-const config = require('../knexfile')[process.env.NODE_ENV || 'development']
+const config = require('../knexfile')[process.env.DB || 'development']
```

Comment:
`process.env.NODE_ENV` is set to 'production' by default, so would probably be the better choice here.